test/odbc/psqlodbc/bigint.cpp

```diff
@@ -0,0 +1,348 @@
+#include <gtest/gtest.h>
+#include <sqlext.h>
+#include "../src/odbc_handler.h"
+#include "../src/query_generator.h"
+#include "../src/drivers.h"
+#include <cmath>
+#include <iostream>
+#include <time.h>
+#include "psqlodbc_tests_common.h"
+using std::pair;
+
+const string TABLE_NAME = "master_dbo.bigint_table_odbc_test";
+const string COL1_NAME = "pk";
+const string COL2_NAME = "data";
+const string DATATYPE_NAME = "sys.bigint";
+const string VIEW_NAME = "master_dbo.bigint_view_odbc_test";
+static vector<pair<string, string>> TABLE_COLUMNS = {
+  {COL1_NAME, "INT PRIMARY KEY"},
+  {COL2_NAME, DATATYPE_NAME}
+};
+
+const int INT_BYTES_EXPECTED = 4;
+const int BIGINT_BYTES_EXPECTED = 8;
+
+class PSQL_DataTypes_BigInt : public testing::Test {
+  void SetUp() override {
+    if (!Drivers::DriverExists(ServerType::PSQL)) {
+      GTEST_SKIP() << "PSQL Driver not present: skipping all tests for this fixture.";
+    }
```

Comment:
This section also needs to be added to the `TearDown()`, something I need to adjust as well.